lighthouse-core/report/html/report-styles.css

```diff
@@ -480,6 +480,7 @@
 
 .lh-audit__display-text {
   flex: 1;
+  word-break: normal;
```

Comment:
I would love to see a diagram of our web of word-break fixes over the course of LH history 😆